indico/web/client/js/react/hooks.js

```diff
@@ -103,10 +103,11 @@ FavoritesProvider.propTypes = {
   children: PropTypes.func.isRequired,
 };
 
-export function useIndicoAxios({camelize, ...args}) {
+export function useIndicoAxios({camelize, unhandledErrors = [], ...args}) {
   const lastData = useRef(null);
   const {response, error, loading, reFetch} = useAxios({
-    customHandler: err => err && handleAxiosError(err),
+    customHandler: err =>
+      err && !unhandledErrors.includes(err.response.status) && handleAxiosError(err),
```

Comment:
can `err.response` be `null` in case the network request failed?

Reply:
I asked the same thing, I didn't want to expand the condition too much. But I guess it is safer to check it.